parser-typechecker/src/Unison/Codebase/SqliteCodebase.hs

```diff
@@ -685,6 +685,20 @@ sqliteCodebase root = do
             cs <- Ops.causalHashesByPrefix (Cv.sbh1to2 sh)
             pure $ Set.map (Causal.RawHash . Cv.hash2to1 . unCausalHash) cs
 
+          lca :: MonadIO m => Branch m -> Branch m -> m (Maybe (Branch m))
+          lca b1 b2 = do
+            eb1 <- isCausalHash (Branch.headHash b1)
+            eb2 <- isCausalHash (Branch.headHash b2)
+            if eb1 && eb2 && False {- todo, remove this -} then do
```

Comment:
We check that the two hashes exist in the codebase before trying the SQL version.

The SQL wizardry is switched off by this `&& False`, but once the SQL wizardry actually does something other than crash, comment out this `&& False`.

Reply:
I removed the `&& False`.